app/views/subscription/index.phtml

```diff
@@ -45,13 +45,24 @@
 						if ($this->onlyFeedsWithError && !$feed->inError()) {
 							continue;
 						}
-						$error = $feed->inError() ? ' error' : '';
-						$empty = $feed->nbEntries() == 0 ? ' empty' : '';
+
+						$error_class = '';
+						$error_title = '';
+						if ($feed->inError()) {
+							$error_class = ' error';
+							$error_title = _t('sub.feed.error');
+						}
+
+						$empty_class = '';
+						$empty_title = '';
+						if ($feed->nbEntries() == 0) {
+							$empty_class = ' empty';
+							$empty_title = _t('sub.feed.empty');
+						}
+						$mute_class = $feed->mute() ? ' mute' : '';
 				?>
-				<li class="item feed<?= $error, $empty, $feed->mute() ? ' mute' : '' ?>"
-					title="<?= strlen($error) > 0 ? _t('sub.feed.error') : '' ?> <?= strlen($empty) > 0 ? _t('sub.feed.empty') : ''	?>"
-					draggable="true"
-					data-feed-id="<?= $feed->id() ?>">
+				<li class="item feed<?= $error_class, $empty_class, $mute_class ?>" title="<?= $error_title, $empty_title	?>"
```

Comment:
```suggestion
				<li class="item feed<?= $error_class, $empty_class, $mute_class ?>" title="<?= $error_title, $empty_title ?>"
```

Minor: tab instead of space